TrigScint/src/TrigScint/TrigScintFirmwareTracker.cxx

```diff
@@ -53,9 +53,20 @@ void TrigScintFirmwareTracker::produce(framework::Event &event) {
         << "TrigScintFirmwareTracker: produce() starts! Event number: "
         << event.getEventHeader().getEventNumber();
   }
+
+  // A is the mis-alignment vector
   ap_int<12> A[3] = {0, 0, 0};
   ap_int<12> LOOKUP[NCENT][COMBO][2];
 
+  // Initialize the LOOKUP table to zero
+  for (int i = 0; i < NCENT; ++i) {
+    for (int j = 0; j < COMBO; ++j) {
+      for (int k = 0; k < 2; ++k) {
+        LOOKUP[i][j][k] = ap_int<12>(0);
```

Comment:
I hope putting this to zero is fine

Reply:
I will clone this branch and check gimme a sec. The logic in the array does mean some default values will be bad, but 0 may be an okay one.